Sources/Rendering/Core/CellPicker/index.js

```diff
@@ -264,30 +276,46 @@ function vtkCellPicker(publicAPI, model) {
     if (locator) {
       // TODO when cell locator will be implemented
     } else if (data.getPolys) {
```

Comment:
this test is no longer needed

Reply:
Which test ? `data.getPolys` or `locator`?

Reply:
data.getPolys